pkg/api/unversioned/helpers.go

```diff
@@ -63,6 +64,52 @@ func LabelSelectorAsSelector(ps *LabelSelector) (labels.Selector, error) {
 	return selector, nil
 }
 
+func ParseToLabelSelector(selector string) (*LabelSelector, error) {
```

Comment:
Please add a comment that these functions should be kept in sync with pkg/labels/selector.go.